doc/classes/PhysicsServer3D.xml

```diff
@@ -1152,16 +1152,16 @@
 			Maximum acceleration for the motor at the axes.
 		</constant>
 		<constant name="G6DOF_JOINT_FLAG_ENABLE_LINEAR_LIMIT" value="0" enum="G6DOFJointAxisFlag">
-			If [code]set[/code] there is linear motion possible within the given limits.
+			If set, there is linear motion possible within the given limits.
```

Comment:
```suggestion
			If set, linear motion is possible within the given limits.
```
Same for angular limit.

Reply:
I don't know if the description is correct though, instinctively it sounds like it should be more akin to "Linear motion is constrained to the given limits." - I don't think this enables linear motion per se, but its limitation. But well, probably out of scope for this PR :)